KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -18,6 +19,12 @@ public sealed class DeliveryClient : IDeliveryClient
         private readonly DeliveryEndpointUrlBuilder _urlBuilder;
 
         private IContentLinkUrlResolver _linkUrlResolver;
+
+        /// <summary>
+        /// Richtext content items processor for content items retrieved using this client.
+        /// </summary>
+        public InlineContentItemsProcessor InlineContentItemsProcessor { get; private set; }
```

Comment:
this will have to be pulled up to the `IDeliveryClient` interface as a part of this https://github.com/Kentico/delivery-sdk-net/pull/42

if you know what I'm talking about you can go ahead and fix it yourself. otherwise, I'll fix it during the merge procedure myself